tests/functional/replication/queueProcessor.js

```diff
@@ -30,19 +31,43 @@ const constants = {
         accessKey: 'accessKey2',
         secretKey: 'verySecretKey2',
         canonicalId: 'bac4bea1bac4bea1bac4bea1bac4bea1bac4bea1bac4bea1bac4bea1bac4bea1',
-        dataKey: '7d808697fbaf9f16fb32b94be189b80b3b9b2890',
+        dataKey: '6d808697fbaf9f16fb32b94be189b80b3b9b2891',
```

Comment:
same (and they can/should actually be different from the source keys, for test purposes it can be interesting to make them different).